custom_addons/legal_compliance_management/views/x_monitoring_measurement_views.xml

```diff
@@ -88,10 +88,11 @@
                     <field name="title_of_programme"/>
                     <field name="test_frequency"/>
                     <field name="test_performed_by"/>
-                    <field name="date_time"/>
-                    <field name="test_result"/>
-                    <field name="upload_test_report"/>
-                    <field name="attachment_ids" widget="many2many_binary" options="{'no_open': True}"/>
+                    <field name="assign_responsibility_id"/>
+                    <field name="test_performed_on" attrs="{'readonly': [('assign_responsibility_id', '=', False)]}" required="True"/>
```

Comment:
Why is condition set to false. we need to check if the current logged in user is same as the user mentioned in assign_responsibility_id right. Try something like this [('assign_responsibility_id.id', '=', uid)] or [('assign_responsibility_id', '=', uid)]